lib/asciidoctor-revealjs/converter.rb

```diff
@@ -0,0 +1,30 @@
+require 'asciidoctor/converter/html5'
+require 'asciidoctor/converter/composite'
+require 'asciidoctor/converter/template'
+
+module Asciidoctor
+module Revealjs
+class Converter < ::Asciidoctor::Converter::CompositeConverter
+  ProvidedTemplatesDir = ::File.expand_path '../../../templates', __FILE__
+  register_for 'revealjs'
+
+  def initialize backend, opts = {}
+      # merge user templates with provided templates (user wins)
+      template_dirs = [ProvidedTemplatesDir]
+      if (user_template_dirs = opts[:template_dirs])
+        template_dirs += user_template_dirs.map {|d| ::File.expand_path d }
+      end
+      # create the main converter
+      template_converter = ::Asciidoctor::Converter::TemplateConverter.new backend,
+          template_dirs,
+          (opts.merge htmlsyntax: 'html', template_engine: 'slim')
```

Comment:
If we want to use the same converter in Asciidoctor.js (cross-compiled with Opal), I think we need to provide an easy way to override this value from `slim` to `jade`